docs: update apt messages on docs (#2445)
canonical/ubuntu-advantage-client: docs/explanations/apt_messages.md

## ESM series with esm-infra service disabled

> When you are running `apt upgrade` on ESM release, like Xenial, we advertise

Reviewer: ```suggestion
When you run `apt upgrade` on an ESM release, like Xenial, we advertise
```
